src/microsim/schema/simulation.py

```diff
@@ -123,6 +129,43 @@ def optical_image(
             raise ValueError("truth must be a DataArray")
         # let the given modality render the as an image (convolved, etc..)
         channel = self.channels[channel_idx]  # TODO
+
+        # get the emission events for the given fluorophore
+        fluorophore_str = "EGFP"
```

Comment:
this can be gleaned from channel, (which is an OpticalConfig), and you can use the `OpticalConfig.excitation` property, which combines all excitation filters into a single "effective" excitation, and the same goes for emission.